### ISSUE TITLE
"Variable is not declared" occurs in :delfunction

### ISSUE THREAD

**Issue by thinca** [Kuniwak/vint#105]
```vim
delfunction Foo  " OK
function! Func() abort
  delfunction Foo  " NG
endfunction
```

```
test.vim:3:15: Variable is not declared (see :help E738)
```

**Comment by Kuniwak** [comment]
Thanks @thinca !

This bug already fixed on #93.
So it will be fiexed by the next release.

**Comment by thinca** [comment]
! Thanks!
